app/views/HasCommodityCodeView.scala.html

```diff
@@ -46,10 +46,10 @@
 
 @detailsContent = {
     <p class="govuk-body">
-         @messages("hasCommodityCode.paragraph.1")
+        @paragraph(Html(messages("hasCommodityCode.paragraph.1")))
```

Comment:
No need to use "\<p>" tag and @paragraph. They seem to do the same things. Use one or the other

Reply:
Done